schema.sql

```diff
@@ -7,5 +7,68 @@ CREATE TABLE animals(
   escape_attempts   INT,
   neutered boolean,
   weight_kg decimal,
-  PRIMARY KEY(id) G
-);
\ No newline at end of file
+  PRIMARY KEY(id) 
+);
+
+ alter table animals add column species varchar(40)
+
+ create table owners(
+	id serial,
+	full_name varchar(40),
+	age int,
+	primary key(id)
+);
+
+select * from owners
+
+create table species(
+	id serial,
+	full_name varchar(40),
+	age int,
+	primary key(id)
+);  
```

Comment:
Great job @r-ahmed2022 🤝

- [ ] As stated in the project requirements, kindly remove the `age` column from the `species` table.
![species](https://user-images.githubusercontent.com/63917725/220841753-41d8f19c-708a-4b24-9998-f81c5c717313.PNG)

- [ ] [Optional] You may as well update `full-name` to `name`.

Reply:
@1GORDON ,  In my opinion, there is no age column in the species table...